src/tests/ftest/io/ior_small.yaml

```diff
@@ -13,6 +13,9 @@ hosts:
 timeout: 500
 server_config:
     name: daos_server
+ server:
+  bdev_class: nvme
+  bdev_list: ["0000:81:00.0"]
```

Comment:
Is it possible to autofill the value of bdev_list? I think "daos_server storage scan" gives us that info?

Reply:
Please refer new PR#1067. I am going to close this one.

Same question was answered in PR#1067